package.json

```diff
@@ -50,6 +50,12 @@
   },
   "nyc": {
     "all": true,
+    "check-coverage": true,
+    "per-file": true,
+    "lines": 100,
+    "statements": 100,
+    "functions": 100,
+    "branches": 100,
```

Comment:
Maybe it would be good to allow for a small percentage of uncovered items? Sometimes things are very hard to test and people might change the code for the worse just to get the 100%. Just food for thought. Might also be OK as-is.

Reply:
Personally, I would like to keep the restrictions to try and maintain a high testing standard, even if many do hate writing tests.

I can see your point of view where it can be seen as negative.

In recent experience in our other platforms, I made changes that didn't fail the test but I found later that I broke the emulator process. This is because the area was not tested before.

In other projects for example Webkit, they have a strong requirement but I don't know if they enforce at the test tooling level.

https://webkit.org/contributing-code/#regression-tests

In summary of the Webkit,
From my point of view, they seem to always want things tested, either for a feature or fix. It does bring up a case where tests not provided can be accepted BUT the user must explain why their changes does not require tests.